### PR TITLE
Add org.laborejo.laborejo

### PR DESCRIPTION
**Pull request by fedelibre**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. I'm using `--socket=system-bus`, otherwise the app won't start. I'm currently trying to run the app on Fedora 33 with Pipewire enabled but I can't hear any audio. I tried following the instructions discussed [here](https://github.com/flatpak/flatpak/issues/1509).
- [X] All assets referenced in the manifest are redistributible by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. I wrote a private email to the developer and I'll send him soon the link to this PR.
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge). Well, the author owns the domain.
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. I will submit the metainfo file upstream once this PR has been reviewed.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by fedelibre** [comment]
bot, build org.laborejo.laborejo

**Comment by nedrichards** [comment]
Needs a screenshot in the appdata I'm afraid.

**Comment by fedelibre** [comment]
Ok, I'll add the single screenshot currently available on laborejo.org just to complete the build here. Hopefully I'll use better pictures later.

**Comment by fedelibre** [comment]
bot, build org.laborejo.laborejo

**Comment by barthalion** [comment]
```
❯ flatpak run org.laborejo.laborejo
Gtk-Message: 08:32:05.460: Failed to load module "canberra-gtk-module"
Gtk-Message: 08:32:05.460: Failed to load module "canberra-gtk-module"
Qt: Session management error: None of the authentication protocols specified are supported
[W][000128464.312609][module-rtkit.c:499 idle_func()] could not make thread realtime: Input/output error
ERROR:template.start:Caught crash in execpthook. Trying too execute atexit anyway
Traceback (most recent call last):
  File "/app/bin/laborejo", line 8, in <module>
  File "/app/bin/qtgui/mainwindow.py", line 79, in __init__
  File "/app/bin/template/qtgui/mainwindow.py", line 92, in __init__
  File "/app/bin/template/qtgui/nsmclient.py", line 326, in __init__
  File "/app/bin/template/qtgui/nsmclient.py", line 458, in announceOurselves
  File "/app/bin/template/engine/session.py", line 116, in nsm_openOrNewCallback
  File "/app/bin/engine/main.py", line 42, in __init__
  File "/app/bin/template/engine/sequencer.py", line 63, in __init__
  File "/app/bin/template/engine/sequencer.py", line 68, in _template_processAfterInit
  File "/app/bin/template/engine/metronome.py", line 66, in __init__
  File "/app/bin/template/engine/sequencer.py", line 472, in __init__
  File "/app/bin/calfbox/metadata.py", line 99, in set_pretty_name
  File "/app/bin/calfbox/metadata.py", line 46, in set_property
  File "/app/bin/calfbox/_cbox2.py", line 187, in do_cmd
  File "/app/bin/calfbox/_cbox2.py", line 182, in do_cmd_on
Exception: Set property key:'http://jackaudio.org/metadata/pretty-name' value: 'Metronome-L' to port 'Laborejo:out_1' was not successful
````

This is as far as I can go; I guess it's because I don't have pipewire-jack?

**Comment by fedelibre** [comment]
I'm not sure, but may you try running qjackctl first and then launch laborejo? (qjackctl should be installed in your system)

**Comment by hfiguiere** [comment]
To get sound you need to start qjackctl (I use `pw-jack qjackctl`) or your favourite jack tool and connect the output.

This is because of this upstream pipewire bug
https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/446

**Comment by hfiguiere** [comment]
(I haven't understood how this app work)

**Comment by fedelibre** [comment]
> (I haven't understood how this app work)

Same for me, until I was pointed from the author to the german manual (he's writing it in german, then he will translate it to english). Google Translate does a decent job (even if notes are translated as "grades" and other minor errors). Check [this link](https://translate.google.com/translate?hl=&sl=de&tl=en&u=https%3A%2F%2Flaborejo.org%2Fdocumentation%2Flaborejo%2Fgerman.html%23_navigation_und_pfeiltasten_arbeiten_mit_der_auswahl).

In a nutshell, use the arrow keys to position the cursor on the desired staff line or space, then enter the note duration using numbers (1, 2, 3, 4, etc.). Playback the music from start using Shift+Space.

**Comment by fedelibre** [comment]
@hfiguiere can you please suggest how should I connect Laborejo to MIDI out in Qjackctl?
I've tried some setups, but it always ends up with Laborejo freezing.

Here's what I see:

laborejo-jack

**Comment by fedelibre** [comment]
bot, build org.laborejo.laborejo

(I need to rebuild the app to test it in a virtual machine running Ubuntu Studio)

**Comment by fedelibre** [comment]
bot, build org.laborejo.laborejo

**Comment by barthalion** [comment]
If it works for you, the only thing I'd add is an error window to show that user needs to have jack running. I know about the message above only because I launched it from the terminal.

**Comment by fedelibre** [comment]
Actually it's not working for me yet. I still have to figure out if no playback is due to pipewire or to some mistake I do when I connect the items in Qjackctl.

It would be much easier if the application made the jack connections automatically, as Rosegarden (flatpakked) does. In fact I can playback in Rosegarden, as long as I start jack and qsynth first.

I'm in touch with the author and I'll let you know if we manage to sort it out.
